addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -331,7 +334,18 @@ export const dateTimeField = {
                 `Control the number of minutes in the time selection. E.g. set it to 15 to work in quarters.`
             ),
         },
+        {
+            label: _t("Show seconds"),
+            name: "showSeconds",
+            type: "boolean",
+            default: true,
+            help: _t(`Displays or hides the seconds in the datetime value.`),
+        },
     ],
+    extractProps: ({ attrs, options }, dynamicInfo) => ({
+        ...dateField.extractProps({ attrs, options }, dynamicInfo),
+        showSeconds: exprToBoolean(options.showSeconds ?? true),
```

Comment:
```suggestion
        showSeconds: exprToBoolean(options.show_seconds, true),
```
In archs, the convention is to snake case attributes/options